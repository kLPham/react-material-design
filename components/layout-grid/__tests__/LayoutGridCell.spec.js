import React from 'react';
import { shallow } from 'enzyme';
import LayoutGridCell from '../layoutGridCell';

describe('<LayoutGridCell />', () => {
    it('should render', () => {
        const layoutGridCell = shallow(
            <LayoutGridCell />,
    );
        expect(layoutGridCell).toMatchSnapshot();
    });
    it('should render with columnSpan', () => {
        const columnSpan = 2;
        const layoutGridCell = shallow(
            <LayoutGridCell columnSpan={columnSpan} />,
    );
        expect(layoutGridCell).toMatchSnapshot();
    });
    it('should render with alignment', () => {
        const alignment = 'top';
        const layoutGridCell = shallow(
            <LayoutGridCell alignment={alignment} />,
    );
        expect(layoutGridCell).toMatchSnapshot();
    });
    it('should render with screenSize', () => {
        const columnSpan = 2;
        const screenSize = 'tablet';
        const layoutGridCell = shallow(
            <LayoutGridCell columnSpan={columnSpan} screenSize={screenSize} />,
      );
        expect(layoutGridCell).toMatchSnapshot();
    });
});
