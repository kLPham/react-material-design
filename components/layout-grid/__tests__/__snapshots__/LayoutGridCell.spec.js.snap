// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<LayoutGridCell /> should render 1`] = `
<div
  className="mdc-layout-grid__cell"
/>
`;

exports[`<LayoutGridCell /> should render with alignment 1`] = `
<div
  className="mdc-layout-grid__cell mdc-layout-grid__cell--span-2 mdc-layout-grid__cell--align-middle"
/>
`;

exports[`<LayoutGridCell /> should render with span 1`] = `
<div
  className="mdc-layout-grid__cell mdc-layout-grid__cell--span-2"
/>
`;
